src/NumericalAlgorithms/DiscontinuousGalerkin/MortarHelpers.cpp

```diff
@@ -22,14 +22,14 @@ Mesh<Dim> mortar_mesh(const Mesh<Dim>& face_mesh1,
   for (size_t i = 0; i < Dim; ++i) {
     ASSERT(
         face_mesh1.basis(i) == Spectral::Basis::Legendre and
-            face_mesh2.basis(i) == Spectral::Basis::Legendre and
-            face_mesh1.quadrature(i) == Spectral::Quadrature::GaussLobatto and
-            face_mesh2.quadrature(i) == Spectral::Quadrature::GaussLobatto,
-        "Only LGL meshes are supported for element faces so far.");
+            face_mesh2.basis(i) == Spectral::Basis::Legendre,
+        "Only Legendre basis meshes are supported for element faces so far.");
     mortar_extents[i] = std::max(face_mesh1.extents(i), face_mesh2.extents(i));
   }
-  return {mortar_extents.indices(), Spectral::Basis::Legendre,
-          Spectral::Quadrature::GaussLobatto};
+  // In 0-d we don't care about basis or quadrature so just specify GaussLobatto
+  return {
+      mortar_extents.indices(), Spectral::Basis::Legendre,
+      Dim != 0 ? face_mesh1.quadrature(0) : Spectral::Quadrature::GaussLobatto};
```

Comment:
I think it would be helpful to document (and therefore test) that the quadrature of "mesh1" gets used if the two meshes disagree. Or instead, should it be checked that both meshes have the same quadrature?

Reply:
I'll add an `ASSERT` requiring the quadrature be equal in all cases. Sound good?